src/components/Avatar.tsx

```diff
@@ -80,18 +80,19 @@ function Avatar({
     }, [originalSource]);
 
     const isWorkspace = type === CONST.ICON_TYPE_WORKSPACE;
-    const iconSize = StyleUtils.getAvatarSize(size);
 
-    const imageStyle: StyleProp<ImageStyle> = [StyleUtils.getAvatarStyle(size), imageStyles, styles.noBorderRadius];
-    const iconStyle = imageStyles ? [StyleUtils.getAvatarStyle(size), styles.bgTransparent, imageStyles] : undefined;
-
-    // We pass the color styles down to the SVG for the workspace and fallback avatar.
-    const source = isWorkspace ? originalSource : UserUtils.getAvatar(originalSource, Number(accountID));
+    // if it's user avatar then accountID will be a number
+    const source = isWorkspace ? originalSource : UserUtils.getAvatar(originalSource, accountID as number);
```

Comment:
@s77rt any better ideas for `accountID as number`?

the logic here is
- if `isWorkspace` is true -> the `accountID` will be a `policyID` which is a `string`
- if `isWorkspace` false -> accountID is standard user account id which is a `number`

I dislike using casting via `as`, but not sure how to improve this line

Reply:
We can split the types into two and use union
```js
type AvatarProps = {
    ...
} & (
    | {
          type: typeof CONST.ICON_TYPE_AVATAR;
          accountID?: number;
      }
    | {
          type: typeof CONST.ICON_TYPE_WORKSPACE;
          accountID?: string;
      }
);
```

However for this to work we need to make the `type` prop required and remove its default value.